packages/openapi-tanstack-query/tsconfig.build.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
```

Comment:
Note: we only need a `tsconfig.build.json` when we want a different output for builds (e.g. when we want to exclude tests, or change the `module`, etc.). We may not need that here; if so just remove it (for now)

Reply:
Same, copy pasted. I usually go for `tsconfig.json` (build), and then `tsconfig.examples.json` (extending the build one), and `tsconfig.lint.json` if necessary.

I'll keep that open to not forget